src/views/catalog/wizard/tabs/disks/components/DiskRowActions.tsx

```diff
@@ -74,12 +74,13 @@ const DiskRowActions: React.FC<DiskRowActionsProps> = ({ diskName }) => {
         isOpen={isOpen}
         obj={vm}
         onSubmit={onDelete}
-        headerText={t('Detach {{diskName}} disk', { diskName })}
+        headerText={t('Detach {{diskName}} disk?', { diskName })}
         submitBtnText={deleteBtnText}
         submitBtnVariant={ButtonVariant.danger}
       >
         <Trans t={t}>
-          Are you sure you want to detach <strong>{diskName} </strong>
+          Are you sure you want to detach <strong>{diskName} </strong>in namespace{' '}
```

Comment:
in file above [DeleteResourceMessage.tsx](https://github.com/kubevirt-ui/kubevirt-plugin/pull/926/files#diff-5170b9ba43e92451f91fe0e0a71393282c4df1b33244ab9ebbf9b9d0768cc1a7) you add a check for namespace but here no? i think better keep on consistency, not sure namespace check is required, as you don't check for name also, they both comes together.. or I'm missing something ?

Reply:
Here we use "detach", so I haven't used `DeleteResourceMessage` component, and it is very highly expected that we display the namespace in this modal, too. 
In `DeleteResourceMessage`, there's check for namespace added, because displaying namespace is redundant for MigrationPolicy deletion. In all other cases, we do want to display the namespace. Also using `vm?.metadata?.namespace` syntax makes sure the app doesn't break in (very rare and unexpected) case the namespace is not present.